pkg/core/mpt/node_test.go

```diff
@@ -74,6 +90,38 @@ func TestNode_Serializable(t *testing.T) {
 	})
 }
 
+func TestInvalidJSON(t *testing.T) {
```

Comment:
Encoding/decoding and invalid JSON tests are all nice, but we also need some known-good JSON strings to decode. The ones generated by encoding are not appropriate as we need to follow a specific encoding standard set by C# implementation and internal enc/dec doesn't prove we're compliant with that.

Reply:
Are there any that can be taken from C# tests or generated by the respective node?

Reply:
Found only this https://github.com/neo-project/neo/blob/neox-2.x/neo.UnitTests/UT_MPTTrie.cs#L198
Added it.